conda/base/context.py

```diff
@@ -889,7 +899,7 @@ def restore_free_channel(self) -> bool:
         return self._restore_free_channel
 
     @memoizedproperty
-    def custom_multichannels(self):
+    def custom_multichannels(self) -> dict[str, Iterable[Channel]]:
```

Comment:
Looks like we can expect it to be a tuple:

```suggestion
    def custom_multichannels(self) -> dict[str, tuple[Channel, ...]]:
```